#goal definition
char[] outputData
char[] Endpoint
---
#result definition
bool isOK
---
#feedback
int32[] sequence
